template/page-dev-tools.html.jinja

```diff
@@ -0,0 +1,193 @@
+{% extends "base.html.jinja" %}
+
+{% block bodyclasses %}no-sidebar{% endblock %}
+{% block mainclasses %}page-dev-tools{% endblock %}
+
+{% block main %}
+
+<div class="">
+  <section class="py-20">
+    <div class="mx-auto text-lg-left">
+      <div class="d-flex flex-column-reverse">
+        <p class="mb-0">
+          {% trans %}Use the developer tools to test, explore, and validate XRP Ledger
+          API requests and behavior. {% endtrans %}
```

Comment:
Let's not hard-wrap user-translatable strings if we can avoid it. Also, please trim out extra space inside the `{% trans %}` blocks.

```suggestion
          {% trans %}Use the developer tools to test, explore, and validate XRP Ledger API requests and behavior.{% endtrans %}
```